service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -38,10 +38,7 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
-import org.jooq.DSLContext;
-import org.jooq.Field;
-import org.jooq.Record5;
-import org.jooq.Result;
+import org.jooq.*;
```

Comment:
Please revert this back to non-wildcard import. See [Intellij docs](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-) since it's set to do this automatically (but it's not great practice).

Reply:
Looks good